docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -18,38 +19,51 @@ The version numbers can be found in these Perl modules:
 Any changes since the last release must be documented in the Changes files.
 Please refer to any Github issues related to the change by the issue number.
 
+ * zonemaster-ldns - [Changes](https://github.com/dotse/zonemaster-ldns/blob/master/Changes)
  * zonemaster-engine - [Changes](https://github.com/dotse/zonemaster-engine/blob/master/Changes)
  * zonemaster-cli - [Changes](https://github.com/dotse/zonemaster-cli/blob/master/Changes)
  * zonemaster-backend - [Changes](https://github.com/dotse/zonemaster-backend/blob/master/CHANGES)
  * zonemaster-gui - [Changes](https://github.com/dotse/zonemaster-gui/blob/master/Changes)
 
 ## 3. Update prerequisites
 
-Make sure the prerequisites section in [README.md](https://github.com/dotse/zonemaster/blob/master/README.md)
-is up to date with regard to [SupportCriteria](https://github.com/dotse/zonemaster/blob/master/docs/internal-documentation/maintenance/SupportCriteria.md).
+Make sure the [declaration of prerequisites](../../../README.md#prerequisites)
+is up to date with regard to [SupportCriteria](SupportCriteria.md).
 
-## 4. Verify that MANIFEST is up to date
+## 4. Update CI configuration
```

Comment:
## 4. Update configuration for Travis checks

Reply:
"CI" has the advantage that it's implementation independent. That way we don't have to change the heading/anchor in case we switch from Travis to some other CI provider.

Reply:
Well, I do not see that problem... We usually talk about "travis" rather than CI or continuous-integration... CI is too cryptic. Then

"Update configuration for continuous-integration"